include/LightGBM/c_api.h

```diff
@@ -283,39 +296,45 @@ LIGHTGBM_C_EXPORT int LGBM_DatasetGetFeatureNames(
 
 
 /*!
-* \brief free space for dataset
-* \return 0 when succeed, -1 when failure happens
+ * @fn LGBM_DatasetFree
+ * @brief Free space for dataset.
+ * @param handle Handle
+ * @return 0 when succeed, -1 when failure happens
 */
 LIGHTGBM_C_EXPORT int LGBM_DatasetFree(DatasetHandle handle);
 
 /*!
-* \brief save dataset to binary file
-* \param handle a instance of dataset
-* \param filename file name
-* \return 0 when succeed, -1 when failure happens
+ * @fn LGBM_DatasetSaveBinary
+ * @brief Save dataset to binary file.
+ * @param handle An instance of dataset
+ * @param filename File name
+ * @return 0 when succeed, -1 when failure happens
 */
 LIGHTGBM_C_EXPORT int LGBM_DatasetSaveBinary(DatasetHandle handle,
                                              const char* filename);
 
 /*!
-* \brief save dataset to text file, intended for debugging use only
-* \param handle a instance of dataset
-* \param filename file name
-* \return 0 when succeed, -1 when failure happens
+ * @fn LGBM_DatasetDumpText
+ * @brief Save dataset to text file, intended for debugging use only.
+ * @param handle An instance of dataset
+ * @param filename File name
+ * @return 0 when succeed, -1 when failure happens
 */
 LIGHTGBM_C_EXPORT int LGBM_DatasetDumpText(DatasetHandle handle,
                                            const char* filename);
 
 /*!
-* \brief set vector to a content in info
-*        Note: group and group only work for C_API_DTYPE_INT32
-*              label and weight only work for C_API_DTYPE_FLOAT32
-* \param handle a instance of dataset
-* \param field_name field name, can be label, weight, group, group_id
-* \param field_data pointer to vector
-* \param num_element number of element in field_data
-* \param type C_API_DTYPE_FLOAT32 or C_API_DTYPE_INT32
-* \return 0 when succeed, -1 when failure happens
+
+ * @fn LGBM_DatasetSetField
+ * @brief Set vector to a content in info.
+ *        Note: group and group only work for C_API_DTYPE_INT32,
+ *              label and weight only work for C_API_DTYPE_FLOAT32.
+ * @param handle An instance of dataset
+ * @param field_name Field name, can be label, weight, group, group_id
+ * @param field_data Pointer to vector
+ * @param num_element Number of element in field_data
+ * @param type C_API_DTYPE_FLOAT32 or C_API_DTYPE_INT32
```

Comment:
` * @param type Type of data pointer, can be C_API_DTYPE_INT8, C_API_DTYPE_INT32, C_API_DTYPE_FLOAT32 or C_API_DTYPE_FLOAT64`